src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -843,6 +819,8 @@ const DEFAULT_VALUES = {
   defaultScore: 0,
   dragMargin: 8,
   selectedDef: null,
+
+  numberOfBins: 32,
```

Comment:
OK, that's reasonable, but we still have 'numberOfBins: 32' scattered in several places. If we decide to, we should be able to update the number of bins for the whole application by changing a single constant. Where does that definition live? I was thinking the provider would have a default, the application could set that default, and the view components could query and use the default numberOfBins.